File: 5_D/src/Kullanici.java
public class Kullanici {
    private String isim,mail,IP;
    private int tel;

    public Kullanici(String isim,String mail, String IP,int tel){
        this.isim=isim;
        this.mail=mail;
        this.IP=IP;
        this.tel=tel;
    }

    public String getIsim(){
        return isim;
    }
    public String getMail(){
        return mail;
    }
    public String getIP(){
        return IP;
    }
    public int getTel(){
        return tel;
    }

    public void setIsim(String isim){
        this.isim=isim;
    }
    public void setMail(String mail){
        this.mail=mail;
    }
    public void setIP(String IP){
        this.IP=IP;
    }
    public void setTel(int tel){
        this.tel=tel;
    }


}

File: 5_D/python/Kullanici.py

class Kullanici:
    def __init__(self,isim=None ,mail=None,ip=None, tel=None):
        self.isim= isim
        self.mail= mail
        self.ip= ip
        self.tel= tel
    
    def getIsim(self):
        return self.isim
    def setIsim(self,isim):
        self.isim=isim

    def getMail(self):
        return self.mail
    def SetMail(self,mail):
        self.mail=mail
    
    def getIp(self):
        return self.ip
    def setIp(self,ip):
        self.ip=ip
    
    def getTel(self):
        return self.tel
    def setTel(self,tel):
        self.tel=tel

